### ISSUE TITLE
Offline QuantumEspresso installation not possible 

### ISSUE THREAD

**Issue by ocaisa** [hpcugent/easybuild-easyblocks#1039]
It turns out that QuantumEspresso has a (new? I'm using 6.0) feature to download the necessary sources for the extensions. Since this is not handled by EB, on an offline node you end up with the following error:
```
*** Unable to download qe-gipaw. Test whether curl or wget is installed and working,
*** if you have direct access to internet. If not, copy into archive/ the file
*** located here http://files.qe-forge.org/index.php?file=qe-gipaw-6.0.tar.gz
```
If your node is online you won't notice this

**Comment by ocaisa** [comment]
Ah, it turns out that gipaw needs to be added as one of the special cases, the directory created is `qe-gipaw` not `gipaw` as the easybock currently expects.

**Comment by ocaisa** [comment]
...and it's an awkward one because it already the `dirname_head` is malformed because the stub already contains a `-`

**Comment by boegel** [comment]
fixed in #1041, thanks @ocaisa
